### ISSUE TITLE
Sync recent SSH remote targets

### ISSUE THREAD

**Issue by ThiefMaster** [microsoft/vscode-remote-release#5369]
When switching e.g. between home and office, it'd be much more convenient to have the same list of recently-used SSH targets (and more important, workspaces/folders on these targets) on all computers.

**Comment by jameszhang-a** [comment]
Do you know where I can find a list of all of the recently used locations? 

![image](https://user-images.githubusercontent.com/83667312/127219488-ef8667e4-9ef0-4ca7-9ccb-e69672c2889f.png)
For example, I'd like to export all of these folders I've visited from this host to another computer, so I can directly connect to them.

**Comment by bamurtaugh** [comment]
Thanks for opening @ThiefMaster. We've previously closed this as out of scope: https://github.com/microsoft/vscode-remote-release/issues/4859#issuecomment-820663415. 

If I've misunderstood any part of the request here, please just let me know. Thanks!

**Comment by ThiefMaster** [comment]
@bamurtaugh yes my request is different. I don't want to sync ssh config but the list of recent folders/workspaces for my remotes

**Comment by bamurtaugh** [comment]
cc @roblourens for thoughts.

**Comment by ThiefMaster** [comment]
If this is not going to be implemented, I'd at least like to know where this is stored so I can manually copy it. This is also relevant when switching e.g. from insiders to release or vice versa - losing the history of recently opened remotes would be quite annoying.

**Comment by lucamaf** [comment]
where is the list of recent ssh connections stored?
